compiler/linter.nim

```diff
@@ -109,7 +109,8 @@ proc differs(conf: ConfigRef; info: TLineInfo; newName: string): string =
   var first = min(info.col.int, line.len)
   if first < 0: return
   #inc first, skipIgnoreCase(line, "proc ", first)
-  while first > 0 and line[first-1] in Letters: dec first
+  if first < line.len and line[first] != '.':
```

Comment:
Why would it be a '.'?

Reply:
For `pcre.Pcre`, it is parsed into dot expr. The col stops at the position of `.`. So the `first` variable need to increase one lest the `.` is accounted.